utils/src/ooinstall/openshift_ansible.py

```diff
@@ -11,30 +11,46 @@
 
 CFG = None
 
+ROLES_TO_GROUPS_MAP = {
+    'master': 'masters',
+    'node': 'nodes',
+    'storage': 'nfs'
+}
+
+VARIABLES_MAP = {
+    'ansible_ssh_user':'ansible_ssh_user',
+    'ansible_config':'ansible_config',
+    'ansible_log_path':'ansible_log_path',
```

Comment:
Any real reason to have the above ansible_* vars as part of a mapping, since the names are the same for input and output?